test/width_test.py

```diff
@@ -0,0 +1,86 @@
+from multidiff.Render import *
+from multidiff import Ansi
+from multidiff.command_line_interface import make_parser, get_max_width
+
+import shutil
+from unittest import TestCase
+import argparse
+
+
+class WidthTests(TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_parser(self):
+        args = make_parser().parse_args([])
+        res = argparse.Namespace(
+            color='ansi', file=[], informat=None,
+            mode='sequence', outformat='hexdump', port=None, stdin=False,
+            width='82'
+        )
+        assert(args == res)
+
+    def test_get_max_width(self):
+        args = make_parser().parse_args(['--width', 'max'])
+        args.width = get_max_width(args)
+        columns = int(shutil.get_terminal_size((120, 30)).columns)
+        assert(args.width == columns)
+
+    def test_width(self):
```

Comment:
This should be split into separate tests after every call to ```assert()``` since it's testing several features at once. THe tests are good though 👍